pkgs/by-name/yc/ycmd/package.nix

```diff
@@ -84,6 +84,16 @@ stdenv.mkDerivation {
     mkdir -p $out/bin
     ln -s $out/lib/ycmd/ycmd/__main__.py $out/bin/ycmd
 
+    ## Work-around CMake/Nix naming of `.so` output
+    ln -s $out/lib/ycmd/ycm_core.cpython-[[:digit:]-][^[:space:]]*-gnu.so $out/lib/ycmd/ycm_core.so
```

Comment:
Since we do not want to break darwin, we should use ``stdenv.hostPlatform.extensions.sharedLibrary`` or make it conditional on linux 😅